input/pagecontent/changelog.md

```diff
@@ -9,7 +9,31 @@ This is the log of changes made to the eHealth Implementation Guide.
 #### Instance operations
 ### Code systems
 - Added missing NPU codes to urn:oid:1.2.208.176.2.1 used in concept maps
+- Updated urn:oid:1.2.208.176.2.4 with 11 SKS codes
```

Comment:
If we want to list all the added codes, I would prefer them in list on one line, eg. "... with 11 SKS codes ("DF00", "DF10"....)"

Reply:
Fixed